### ISSUE TITLE
Add epoch metric

### ISSUE THREAD

**Issue by rolfyone** [ethereum/beacon-metrics#10]
It's useful to have slot vs. head_slot, and in a similar way, it's useful to have epoch vs. finalized, justified, previous_justified.

I think we should add epoch to the standard list of metrics...

**Comment by staccDOTsol** [comment]
I completely agree with you. Adding the epoch metric to the standard list of metrics would indeed be useful. It would provide valuable information about the current epoch, allowing for better analysis and monitoring of the Ethereum network. Thank you for suggesting this addition!
